### ISSUE TITLE
[FEATURE REQUEST] Friend List SORTING

### ISSUE THREAD

**Issue by hankgames** [tfedor/AugmentedSteam#1393]

**Comment by candela97** [comment]
Yeah more sorting options is always nice to have, but from your list I only think sorting in alphabetical order is doable at the moment. Steam level requires fetching miniprofile hover content, and age of account requires fetching the badges page for each friend all at once to sort correctly, which would amount to a lot of requests; as for friend added date, I don't know if that info is available anywhere...

**Comment by candela97** [comment]
Yeah more sorting options is always nice to have, but from your list I only think sorting in alphabetical order is doable at the moment. Steam level requires fetching miniprofile hover content, and age of account requires fetching the badges page for each friend all at once to sort correctly, which would amount to a lot of requests; as for friend added date, I don't know if that info is available anywhere apart from the trade offer window...
